force-app/main/default/classes/EinsteinVisionController.cls

```diff
@@ -193,75 +196,77 @@ global with sharing class EinsteinVisionController {
-        req.setMethod('POST');    
+        req.setBodyAsBlob(formBlob);
+        req.setMethod('POST');
         req.setEndpoint(VISION_API + '/train');
         req.setHeader('Authorization', 'Bearer ' + accessToken);
-		req.setHeader('Connection', 'keep-alive');
-		req.setHeader('Content-Length', contentLength);
+        req.setHeader('Connection', 'keep-alive');
+        req.setHeader('Content-Length', contentLength);
         req.setHeader('Content-Type', contentType);
-		req.setHeader('Cache-Control', 'no-cache');
-		req.setTimeout(120000);
+        req.setHeader('Cache-Control', 'no-cache');
+        req.setTimeout(120000);
 
-		try {
-	        Http http = new Http();
+        try {
+            Http http = new Http();
             if (!Test.isRunningTest()) {
-	            HTTPResponse res = http.send(req);
-				return res.getBody();
+                HTTPResponse res = http.send(req);
+                return res.getBody();
             } else {
                 return null;
             }
-        } catch(Exception ex){
-            return '{"error": "' + ex.getMessage() + '"}';
+        } catch (Exception ex) {
+            return '{\"error\": \"' + ex.getMessage() + '\"}';
         }
     }
-    
+
     private static List<Prediction> predictInternal(String sample, String access_token, String model, boolean isBase64) {
         string contentType = HttpFormBuilder.GetContentType();
         //  Compose the form
         string form64 = '';
-        
+
         form64 += HttpFormBuilder.WriteBoundary();
         form64 += HttpFormBuilder.WriteBodyParameter('modelId', EncodingUtil.urlEncode(model, 'UTF-8'));
         form64 += HttpFormBuilder.WriteBoundary();
-        if(isBase64) {
+        if (isBase64) {
             form64 += HttpFormBuilder.WriteBodyParameter('sampleBase64Content', sample);
         } else {
             form64 += HttpFormBuilder.WriteBodyParameter('sampleLocation', sample);
         }
         form64 += HttpFormBuilder.WriteBoundary(HttpFormBuilder.EndingType.CrLf);
-        
+
         blob formBlob = EncodingUtil.base64Decode(form64);
         string contentLength = string.valueOf(formBlob.size());
         //  Compose the http request
         HttpRequest httpRequest = new HttpRequest();
-        
+
         httpRequest.setBodyAsBlob(formBlob);
         httpRequest.setHeader('Connection', 'keep-alive');
         httpRequest.setHeader('Content-Length', contentLength);
         httpRequest.setHeader('Content-Type', contentType);
         httpRequest.setMethod('POST');
         httpRequest.setTimeout(120000);
-        httpRequest.setHeader('Authorization','Bearer ' + access_token);
+        httpRequest.setHeader('Authorization', 'Bearer ' + access_token);
         httpRequest.setEndpoint(VISION_API + '/predict');
-        
+
         Http http = new Http();
         List<Prediction> predictions = new List<Prediction>();
+        // Advance to the start object marker to
+        //  find next probability object.
```

Comment:
Seems like there is an extra space here

Reply:
I think this is a bigger issue. These comments were taken from somewhere else further down the code. I think the association of comments to AST nodes might be off.